bin/src/command/orders.rs

```diff
@@ -611,6 +611,29 @@ impl CommandServer {
             client_id, request_id, id
         );
 
+        if self
```

Comment:
I think this condition will be easier to read, if you opposite like this :

```rust
if self.workers.iter(|worker| worker.id == id && (worker.run_state == RunState::Stopping || worker.run_state == RunState::Stopped)).is_some() {
...
}
```

Reply:
The condition you suggest only checks the aliveness of _existing_ workers. In case of an _unexisting_ worker id, the `find()` method returns None and the main process still crashes!